### PR TITLE
Add papaa package bioconda recipe 

### PR DESCRIPTION
**Pull request by nvk747**
This PR is to add a recipe for the conda package called "papaa". The following package is used for measuring aberrant pathway activity in TCGA pan-cancer dataset using a machine learning approach.  
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by nvk747** [comment]
@BiocondaBot please add label

**Comment by nvk747** [comment]
@BiocondaBot This has been ready for review and merge..did anyone got a chance to look at this?

File: recipes/papaa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/nvk747/papaa/archive/papaa-{{ version }}.tar.gz
+  sha256: c117e9296cf43ff10490449280d61b4da60963e23d146a50fcc9fbc7c636c430
+
+requirements:
+  run:
+    - python==3.6.5
```
Comment (dpryan79) [inline recipes/papaa/meta.yaml:17]: Does it really need exactly python 3.6.5? So 3.7 through 3.9 don't work? Likewise, does numpy >=1.14.3 not work for some reason?

**Review by dpryan79 — APPROVED** [review]

**Comment by nvk747** [comment]
@dpryan79  Thank you for taking a look and approving my changes. I was testing with several combinations and finally could get it worked. I appreciate your effort in merging this PR to bioconda. regards Vijay

**Pull request merged**